ui/app/components/clients/page/token.hbs

```diff
@@ -128,13 +128,10 @@
             />
           {{/unless}}
           {{#if this.byMonthActivityData}}
-            <Clients::RunningTotal
-              @selectedAuthMethod={{this.selectedAuthMethod}}
+            <Clients::Token::MonthlyNew
               @byMonthActivityData={{this.byMonthActivityData}}
-              @isHistoricalMonth={{and (not this.isCurrentMonth) (not this.isDateRange)}}
-              @isCurrentMonth={{this.isCurrentMonth}}
+              @authMount={{this.selectedAuthMethod}}
```

Comment:
Should `this.selectedAuthMethod` be updated to `@authMount`? I didn't catch where the former was defined

Reply:
This will be updated as part of the move to the `counts.token` route since we will be getting that property from the parent controller.